systematics/rawyields/PlotMultiTrialRMSvsPt.py

```diff
@@ -59,7 +67,21 @@
 axisSoverB.SetMaxDigits(3)
 axisSoverB.Draw()
 hSoverB.DrawCopy('same')
+cRMS.Update()
 
-cRMS.SaveAs(args.outFileName)
+cShift = TCanvas('cShift', '', 800, 800)
+cShift.DrawFrame(hMeanShift.GetBinLowEdge(1), 0.1, ptMax, 20., ';#it{p}_{T} (GeV/#it{c}); Mean Shift (%)')
+hMeanShift.DrawCopy('same')
+cShift.Update()
+
+cSyst = TCanvas('cSyst', '', 800, 800)
```

Comment:
if the purpose is to compare RMS, shift and sum in quadrature of the two, why not plotting them in a single panel with a legend?

Reply:
This was just to extract the RMS, shift value and total systematic uncertainties in all pt bins

Reply:
Yes, the point is that typically we quote only the RMS as uncertainty (and that's why only the RMS was displayed), apart from pathological situations. However it might be useful to have a comparison between the three, also to see which is the dominant contribution. Having them separated instead is not very meaningful

Reply:
Done